tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs

```diff
@@ -364,6 +364,58 @@ public void ShowSchedule_PlayoffScheme_RoundNamesAreCreated()
             CollectionAssert.AreEqual(actual.RoundNames, expected.RoundNames);
         }
 
+        /// <summary>
+        /// Test for ShowSchedule method.
+        /// Valid schedule is passed, no exception occurred.
+        /// </summary>
+        [TestMethod]
+        public void ShowSchedule_PlayoffScheme_TeamsNamesAreAssigned()
+        {
+            // Arrange
+            const int TEST_ROUND_COUNT = 6;
```

Comment:
why int?

Reply:
Changing to byte.

Reply:
It seems that it doesn't matter.
![image](https://user-images.githubusercontent.com/11539042/36934100-a1e9d52c-1eec-11e8-959f-1379e6820153.png)